lib/ts/recipe/session/accessToken.ts

```diff
@@ -31,21 +32,44 @@ export async function getInfoFromAccessToken(
     timeCreated: number;
 }> {
     try {
-        verifyJWT(jwtInfo, jwtSigningPublicKey);
-        const payload = jwtInfo.payload;
+        // From the library examples
+        const { payload } = await jose.jwtVerify(jwtInfo.rawTokenString, jwks).catch(async (error) => {
+            // We only want to opt-into this for V2 access tokens
+            if (jwtInfo.version === 2 && error?.code === "ERR_JWKS_MULTIPLE_MATCHING_KEYS") {
```

Comment:
what about version < 2?

Reply:
Since V1 and V2 are functionally the same, we never have version = 1